src/Kdyby/Redis/Driver/PhpRedisDriverOld.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Redis\Driver;
+
+use Kdyby;
+use Nette;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class PhpRedisDriverOld extends \Redis implements Kdyby\Redis\IRedisDriverOld
+{
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function connect($host, $port = NULL, $timeout = 0)
+	{
+		$args = func_get_args();
+		return @call_user_func_array('parent::connect', $args); // intentionally @ - from v3.1.0 Redis::connect() throws warning on failed
```

Comment:
Shouldn't the `@` be in the other class as well then?

Reply:
Do you think `PhpRedisDriver.php` class?
@forrest79 What do you think?

Reply:
According to this https://github.com/phpredis/phpredis/issues/1273 and this `Return real connection error as exception [5b9c0c60]` from changelog http://pecl.php.net/package-changelog.php?package=redis is `@` in `PhpRedisDriver.php` not necessery, because from 4.0.0 is thrown an exception.

Reply:
Hi, I'm not sure, we're using this version with @ in out fork, but it's not necessary for this PR, I just create it on our branch with this, it's possible to remove it here. There is another PR #70 only with this :-)